util/shlib_wrap.sh.in

```diff
@@ -1,5 +1,16 @@
 #!/bin/sh
 
+# To test this OpenSSL version's applications against another version's
+# shared libraries, simply set
+#
+#     OPENSSL_REGRESSION=/path/to/other/OpenSSL/build/tree
+if [ -n "$OPENSSL_REGRESSION" -a \
+        -f "$OPENSSL_REGRESSION/util/shlib_wrap.sh" ]; then
```

Comment:
Wouldn't it be more appropriate to check for -x? Secondly one can wonder if it would be appropriate to handle the error instead of ignoring. I mean if you set OPENSSL_REGRESSION and there is nothing there, you might appreciate an error message that there is nothing there, rather than silent execution with shared libraries in current build tree. Otherwise you give an impression that it regression test succeeded.

Reply:
Good points.  Will fix.